README.md

```diff
@@ -31,43 +31,115 @@ To do this, you must copy the following files in the init directory :
+| directory-notification-server |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| directory-server              |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| dynamic-simulation-server     |        |         |   🗸   |             |        |  🗸    |        |        |         |
+| explore-server                |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| geo-data-server               |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| gridexplore-app               |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| network-map-server            |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| network-modification-server   |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| odre-server                   |        |         |   🗸   |             |        |  🗸    |        |        |         |
+| security-analysis-server      |        |         |   🗸   |             |        |  🗸    |        |        |         |
+| sensitivity-analysis-server   |        |         |   🗸   |             |        |  🗸    |        |        |         |
+| shortcircuit-server           |        |         |   🗸   |             |        |  🗸    |        |        |         |
+| single-line-diagram-server    |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| study-notification-server     |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| study-server                  |        |         |   🗸   |      🗸      |        |  🗸    |        |        |         |
+| timeseries-server             |        |         |   🗸   |             |        |  🗸    |        |        |         |
+| voltage-init-server           |        |         |   🗸   |             |        |  🗸    |        |        |         |
+| case-import-server            |        |         |       |             |        |       |   🗸    |        |         |
+
+To use a profile, you use simply:
+```shell
+$ cd docker-compose
+$ docker compose --profile suite <cmd>
+```
+
+You can also combine multiple profiles:
+```shell
+$ docker compose --profile study --profile dynmap <cmd>
+```
+
+> [!IMPORTANT]  
+> Contrary to what the Docker documentation say:
+>   * With the commands `up` & `down`, services/container who belongs to at least one profile can't be accessed if the profile isn't specified.
+>     For example `docker compose up study-server` would not work because the profile `study` isn't passed in the CLI.
+>     The correct CLI would be `docker compose --profile study up study-server`.
+>   * With the commands `start`, `stop`, `restart`, the `--profile ...` as no effect because theses commands affect the containers already created by a previous `up` command.
```

Comment:
```suggestion
>   * With the commands `start`, `stop`, `restart`, the `--profile ...` has no effect because theses commands affect the containers already created by a previous `up` command.
```